test/apps/iodemo/io_demo.cc

```diff
@@ -1043,6 +1045,7 @@ class DemoClient : public P2pDemoCommon {
             validate(msg, opts().iomsg_size);
         }
 
+        // Client can receive IO_WRITE_COMP or IO_READ_COMP only
```

Comment:
then 
```
if (IO_WRITE_COMP) {
} else {
    assert(IO_READ_COMP)
}
```
should be enough?

Reply:
can fix it with other comments (imo, seems to be more clear this way)

Reply:
ok, i don't have other comments